apps/docs/docs/cdk/sub-modules/template-management/template-management.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_label: 'Template Management'
+sidebar_position: 1
+title: '@rx-angular/cdk/template overview'
+hide_title: true
+---
+
+import Readme, { toc as ReadmeToc } from '@site/../../libs/cdk/template/Readme.md';
```

Comment:
Maybe we should change the file name to README.md instead of changing the reference to Readme.md to conform with the [GNU Coding Standards](https://www.gnu.org/prep/standards/html_node/Releases.html#Releases) ?

Reply:
I was concerned about hard links to those files in the repo and how they are bundled with the packages 🤔

I only see explicit mentions in other Markdown documents, so go ahead and rename all `Readme.md` files to `README.md`. Verify the casing of implicit dependencies in `nx.json` as this update is performed.